_component_design/modal.md

```diff
@@ -46,23 +46,41 @@ helix-ui-javascript: false
 
 <section class="static-section" markdown="1">
 
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
```

Comment:
This probably should span across the whole page as it does not have an image next to it. Remove lines 47-51 to do so.